js/repeater.js

```diff
@@ -546,9 +546,21 @@
 		runRenderer: function(container, renderer, data, callback){
 			var self = this;
 			var skipNested = false;
+			var index = 0;
 			var repeat, subset, i, l;
 
-			var loopSubset = function(index){
+			function loopSubset(){
```

Comment:
I changed it because I read some studies showing that it was potentially *slightly* faster and with the amount of function calls happening in this recursive approach it might be worth it